chapters/unitexpressions.tex

```diff
@@ -25,10 +29,10 @@ \section{The Syntax of Unit Expressions}\doublelabel{the-syntax-of-unit-expressi
 \end{lstlisting}
 
 The unit of measure of a dimension free quantity is denoted by "1". The
-ISO standard does not define any precedence between multiplications and
-divisions. The ISO recommendation is to have at most one division, where
+SI standard does not define any precedence between multiplications and
+divisions. The SI recommendation is to have at most one division, where
 the expression to the right of "\lstinline!/!" either contains no multiplications or
-is enclosed within parentheses. It is also possible to use negative
+is enclosed within parentheses, or use negative
```

Comment:
The reason for the change was that the SI-standard recommends this as well (and "recommends" is a bit vague - it states that you should use one of these alternatives).

One possibility would be ", or use negative ..." -> "; or avoid division by using negative ..."
Another would be to a more complete rewriting:
"The SI standard does not allow multiple units to the right of the division-symbol (\lstinline!/!) since the result is ambigous; either the divisor shall be enclosed in parentheses, or negative exponents used instead of division, for example ..."

Reply:
Both variants look good to me.

Reply:
Selected the more complete rewriting.